pagopa/io-app-design-system
# feat: [IOBP-223] Import new `keyboard` and `contactless` icons

## Short description
This PR adds assets and component icons for the new `keyboard` and `contactless` icons

## List of changes proposed in this pull request
- Added original svg files and react components;
- Mapped `keyboard` and `contactless` icons into `Icon.tsx` file;

## How to test
Open the example app and check if there are [these two new icons](https://www.figma.com/file/SuYjyjZlieQqRS8n1B6qbF/Flussi-App-ID-Pay?type=design&node-id=9611-30793&mode=design&t=65jYXldcf1BenJSs-4)
